### PR TITLE
P.M-A - Issue 1585

### LINKED ISSUE #1585: Add page to display public request details
Breakout from #704

Add a simple page which displays a request's status and (if available) planned deployment date.
No private info (names or addresses etc) should be shown on this page.

### PR DESCRIPTION
**Pull request by pmarshallandrew**
Fixes #1585 
Implemented functionality to display request status information , detailing status and planned deployment date . Added RequestStatus Controller,View, QueryHandler as well as accompanying Unit Tests both for handler and controller.

### CODE REVIEW

File: AllReadyApp/Web-App/AllReady.UnitTest/Controllers/RequestStatusControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace AllReady.UnitTest.Controllers
+{
+    public class RequestStatusControllerTests
+    {
+        [Fact]
+        public async void NonMatchingRequestId_IndexReturnsNotFoundView() {
```
Comment (BillWagner) [inline AllReadyApp/Web-App/AllReady.UnitTest/Controllers/RequestStatusControllerTests.cs]: Can you change the return type from `void` to `Task`?  That way, the test runner awaits the completion of the task before declaring success or failure.

Comment (pmarshallandrew) [inline AllReadyApp/Web-App/AllReady.UnitTest/Controllers/RequestStatusControllerTests.cs]: Apologies in the delay in getting this fixed up and updating you as have been away, ill get this fixed, thank you.

**Comment by tonysurma** [comment]
FYI the appveyor issue is a 'random' issue they are having with a feed and not with your build.  I had the same on master recently.  can you try editing appveyor.yml to change this line `account_feed: true` to `account_feed: false` and see if it works in your PR?  Thanks!

**Comment by tonysurma** [comment]
@pmarshallandrew please rebase to pick up my fix for the appveyor issue and you should be all set.  Thanks for your patience

**Comment by pmarshallandrew** [comment]
@tonysurma   apologies for the delay , built this time :)

**Pull request merged**